appcues/src/main/java/com/appcues/trait/TraitKoin.kt

```diff
@@ -1,19 +1,22 @@
 package com.appcues.trait
 
 import com.appcues.di.KoinScopePlugin
+import com.appcues.trait.appcues.BackdropKeyholeTrait
 import com.appcues.trait.appcues.BackdropTrait
 import com.appcues.trait.appcues.BackgroundContentTrait
 import com.appcues.trait.appcues.CarouselTrait
-import com.appcues.trait.appcues.KeyholeTrait
 import com.appcues.trait.appcues.ModalTrait
 import com.appcues.trait.appcues.PagingDotsTrait
 import com.appcues.trait.appcues.SkippableTrait
 import com.appcues.trait.appcues.StepAnimationTrait
 import com.appcues.trait.appcues.StickyContentTrait
+import com.appcues.trait.appcues.TargetElementTrait
+import com.appcues.trait.appcues.TargetRectangleTrait
 import org.koin.dsl.ScopeDSL
 
 internal object TraitKoin : KoinScopePlugin {
 
+    @Suppress("LongMethod")
```

Comment:
does it make sense to suppress at config level for any `*Koin` files that are just doing these scope declarations?